components/rhs_comment.jsx

```diff
@@ -441,6 +443,32 @@ export default class RhsComment extends React.Component {
             hour12: !this.props.useMilitaryTime
         };
 
+        const messageWrapper = (
+            <PostMessageContainer
+                post={post}
+                isRHS={true}
+                hasMention={true}
+            />
+        );
+
+        const hasPlugin = post.type && this.props.pluginPostTypes.hasOwnProperty(post.type);
+
+        let messageWithAdditionalContent;
+        if (this.props.post.state === Posts.POST_DELETED || hasPlugin) {
+            messageWithAdditionalContent = messageWrapper;
+        } else {
+            messageWithAdditionalContent = (
+                <PostBodyAdditionalContent
+                    post={post}
+                    previewCollapsed={this.props.previewCollapsed}
+                    previewEnabled={this.props.previewEnabled}
+                    isEmbedVisible={this.props.isEmbedVisible}
+                >
+                    {messageWrapper}
+                </PostBodyAdditionalContent>
+            );
+        }
+
```

Comment:
This code was stolen wholesale from [post_body.jsx|https://github.com/mattermost/mattermost-webapp/blob/master/components/post_view/post_body/post_body.jsx#L221]. Looks like we just forgot to handle this case when implementing post previews in the RHS